### ISSUE TITLE
File.blob api

### ISSUE THREAD

**Issue by Chimrod** [ocsigen/js_of_ocaml#292]
Hello,

The signature for the method slicing in File.blob is :

```ocaml
method slice : int -> int -> blob Js.meth
```

where any other methods in the File API expect 
```ocaml
 #File.blob Js.t
```
blocking the following code  :
```ocaml
  let _ = File.readAsBinaryString(file##slice(1, 15))
```
Is this intended or is it a problem in the api ? (Actualy adding some magic make the code works…)

Thanks a lot

**Comment by vouillon** [comment]
Indeed, there was a bug here. It has been fixed just today by commit eb7cae084c3a5e0bd5f534e049fc8698d39836c6.
Thanks for your report!

**Comment by Chimrod** [comment]
What a reactivity ; I didn't expect so much !

Thank you for all the work done with this library :-)
